mne/preprocessing/ica.py

```diff
@@ -2662,10 +2666,11 @@ def read_ica_eeglab(fname):
         # sphere = eye(urchans), so let's use SVD to get our square
         # weights matrix (u * s) and our PCA vectors (v) back
         u, s, v = _safe_svd(eeg.icaweights, full_matrices=False)
-        ica.unmixing_matrix_ = u * s
+        ica.unmixing_matrix_ = u @ s
```

Comment:
`s` is 1D so after this operation `unmixing_matrix_` will be 1D. I think this can't be correct.

Maybe you mean `u @ np.diag(s)`, but this is the same as `u * s`.

Reply:
For more info see previous comment 

https://github.com/mne-tools/mne-python/issues/7657#issuecomment-618040530